Mathlib/Computability/DFA.lean

```diff
@@ -163,7 +174,50 @@ theorem pumping_lemma [Fintype σ] {x : List α} (hx : x ∈ M.accepts)
   rw [Set.mem_singleton_iff] at ha' hc'
   substs ha' hc'
   have h := M.evalFrom_of_pow hb hb'
-  rwa [mem_accepts, evalFrom_of_append, evalFrom_of_append, h, hc]
+  rwa [mem_accepts', evalFrom_of_append, evalFrom_of_append, h, hc]
 #align DFA.pumping_lemma DFA.pumping_lemma
 
+section Maps
+
+universe u' v'
+variable {α' : Type u'} {σ' : Type v'} (f : α ≃ α') (g : σ ≃ σ') (s : σ') (a : α') (x : List α')
```

Comment:
No need specifying universes unless you're doing category theory
```suggestion
section Map
variable {α' σ' : Type*} (f : α ≃ α') (g : σ ≃ σ') (s : σ') (a : α') (x : List α')
```

Reply:
Done. It might be worth removing the existing universe declarations, but that can be done in another PR